openquake/server/dbserver.py

```diff
@@ -34,7 +34,7 @@
 from openquake.server import dbapi
 from openquake.server.settings import DATABASE
 
-executor = ProcessPoolExecutor(1)  # there is a single db process
+executor = ThreadPoolExecutor(1)  # there is a single db process
 
```

Comment:
Please replace the comment `# there is a single db process` with something like `# using a ThreadPool because fork and sqlite3 do not play well on macOS Sierra`